src/utils/useScrolled.ts

```diff
@@ -1,14 +1,14 @@
 import { useState, useEffect } from "react";
 
-/* Returns if page is scrolled past the header. */
-export function useScrolledHeader() {
+/* Returns if page is scrolled past given element. */
+export function useScrolled(elementId) {
   const [isScrolled, setIsScrolled] = useState(true);
 
   useEffect(() => {
     function handleScroll() {
-      const header = document.getElementById("header");
-      if (header) {
-        const offset = header.offsetHeight;
+      const element = document.getElementById(elementId);
+      if (element) {
+        const offset = element.offsetHeight + 300;
```

Comment:
Can you make 300 a variable such as `DELAY_STICKY_TRANSITION` (based on the comments in the ticket). Just to keep a history of where this number is coming.